src/library/legacylibraryimporter.cpp

```diff
@@ -86,23 +87,23 @@ void LegacyLibraryImporter::import() {
         QDomNode track;
 
         for (int i = 0; i < trackList.size(); i++) {
-            //blah, can't figure out how to use an iterator with QDomNodeList
+            // blah, can't figure out how to use an iterator with QDomNodeList
```

Comment:
this piece of code is not mine, I only "own" the space between // and the comment ;)
I will not refactor this in this PR as it is unrelated with the PR object.